lib/common-args.js

```diff
@@ -73,8 +73,7 @@ module.exports = function commonArgs(app) {
     .option('timeout', {
       alias: 'o',
       type: 'number',
-      description: 'Set timeout for npm install',
-      default: 1000 * 60 * 10
```

Comment:
Should the default not be kept / how come the removal is necessary?

Reply:
If the default is set at this point it overrides the module-specific timeout later on at https://github.com/nodejs/citgm/pull/740/files#diff-166959275b22802c268f62f152801cbcR9.

If `--timeout` is not provided, it falls back to `kDefaultTimeout` (which previously seems to have been redundant).